docs/spark/spark-writes.md

```diff
@@ -328,14 +330,32 @@ PARTITIONED BY (days(ts), category)
 
 To write data to the sample table, your data needs to be sorted by `days(ts), category`.
 
-If you're inserting data with SQL statement, you can use `ORDER BY` to achieve it, like below:
+If you're inserting data with SQL statement, you can use either `ORDER BY` to trigger global sort, or `SORT BY` to trigger local sort. Global sort like below:
```

Comment:
Since we are changing this line let's drop the pronouns. 

I think the examples may be clearer as well if we just name them explicitly

"To globally sort data based on ts and category
Example
"